### PR TITLE
add lb/ip blocks for eu-west3 and us-west4 | OPS-16210

### PR DESCRIPTION
**Pull request by lukmdo**
- [preview](https://github.com/saucelabs/sauce-docs/blob/ops-16210-update-public-ip-blocks/docs/basics/data-center-endpoints.md)
- [main docs](https://docs.saucelabs.com/basics/data-center-endpoints/)

### Description
Add new IP blocks for GKE dc (EU, and US-west).

It is to help with changes coming in [OPS-16210](https://saucedev.atlassian.net/browse/OPS-16210) and customer communication. 

Merge once got representative signoff.  

### Motivation and Context
Some customers who do IP white/blacklist their network should include new IP blocks ( and later clean-up old IP blocks  - see comments ) 

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist
- [x] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/basics/data-center-endpoints.md [hunk 1]
```diff
@@ -137,7 +137,7 @@ Depending on the framework or driver you use, you might need to make additional
 <td>*.tunnels.eu-west-3-lnbf.saucelabs.com</td><td><code>34.141.28.96/32</code></td>
 </tr>
 <tr>
-<td>Other endpoints</td><td><code>185.94.24.0/22</code></td>
+<td>Other endpoints</td><td><code>34.107.82.96/27</code><br/><code>185.94.24.0/22</code></td>
```
Comment (lukmdo) [inline docs/basics/data-center-endpoints.md:140]: context 

> ondemand.eu-central-1.saucelabs.com has address 185.94.26.249

old 185.* IP block will be deprecated (EOL date??) and switch to new IP block as above

File: docs/basics/data-center-endpoints.md [hunk 2]
```diff
@@ -85,7 +85,7 @@ URLs vary based on the services you're using, your data center, and the types of
 <td>*.miso.saucelabs.com</td><td><code>66.85.48.0/21</code></td>
 </tr>
 <tr>
-<td>Other endpoints</td><td><code>162.222.72.0/21</code><br/><code>66.85.48.0/21</code></td>
+<td>Other endpoints</td><td><code>34.125.90.96/27</code><br/><code>162.222.72.0/21</code><br/><code>66.85.48.0/21</code></td>
```
Comment (lukmdo) [inline docs/basics/data-center-endpoints.md:88]: context 

> ondemand.us-west-1.sacuelabs.com has address 185.53.179.171
> ondemand.sacuelabs.com has address 185.53.179.171
> jpmc.ondemand.saucelabs.com has address 66.85.49.22

`66.85.*` and `185.53.*` will be deprecated (EOL date??) and replaced by the new IP block added

**Review by lauramthomas — APPROVED** [review]

File: docs/basics/data-center-endpoints.md (hunk 2 above)
Comment (lukmdo) [reply docs/basics/data-center-endpoints.md:91]: I believe that DNS cutover will happen on 10th October

**Comment by lukmdo** [comment]
see [#coreservices-slack thread|https://saucelabs.slack.com/archives/C0L0J10V7/p1666808125549759] for some more info

**Comment by lysannep** [comment]
@lauramthomas Can you help with this PR?

**Pull request closed without merging**